### PR TITLE
feat: Add a `claim-rewards` command

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [x]  Targeted PR against correct branch
- [x]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x]  Linked to Github issue with discussion and accepted design
- [x]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

File: cmd/tx/claim/claim.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+				Denom:  consts.Denoms.Hub,
+				RPC:    mainnetHub.RPC_URL,
+			}, sequencerAddr)
+			if err != nil {
+				return err
+			}
+			// Calculated by sending a tx on Froopyland and see how much fees were paid
+			txGasPrice := big.NewInt(50000)
+			totalBalanceMinusFees := new(big.Int).Sub(sequencerBalance.Amount, txGasPrice)
+			if totalBalanceMinusFees.Cmp(big.NewInt(0)) != 1 {
```
Comment (mtsitrin) [inline cmd/tx/claim/claim.go:48]: can be negative?

File: cmd/tx/claim/claim.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+				return fmt.Errorf("no rewards to claim for the address associated with the given private key: %s"+
+					"please try to import the private key to keplr and claim the rewards from there",
+					sequencerAddr)
+			}
+			rewardsAmountStr := totalBalanceMinusFees.String() + consts.Denoms.Hub
+			sendAllFundsCmd := exec.Command(consts.Executables.Dymension, "tx", "bank", "send", consts.KeysIds.HubSequencer,
+				args[1], rewardsAmountStr, "--node", mainnetHub.RPC_URL, "--chain-id", mainnetHub.ID,
+				"--fees", txGasPrice.String()+consts.Denoms.Hub, "-b", "block", "--yes", "--home", tempDir)
```
Comment (mtsitrin) [inline cmd/tx/claim/claim.go:56]: u sure the `--yes` flag is required and good practice?

File: cmd/tx/claim/claim.go (hunk 1 above)
Comment (ItayLevyOfficial) [reply cmd/tx/claim/claim.go:48]: If the returned value is not 1 then the balance is <=0

File: cmd/tx/claim/claim.go (hunk 2 above)
Comment (ItayLevyOfficial) [reply cmd/tx/claim/claim.go:56]: Ye - otherwise the tx flow will be interactive and fail

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**